File: Ser2023/Company/Company.Web/Modules/Company/BusinessEntity/BusinessEntityColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace Company.Company.Columns;

[ColumnsScript("Company.BusinessEntity")]
[BasedOnRow(typeof(BusinessEntityRow), CheckNames = true)]
public class BusinessEntityColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int BusinessEntityId { get; set; }

    //[EditLink, Width(60)]
    //public string PersonType { get; set; }
    //public string Title { get; set; }
    //public string FirstName { get; set; }
    //public string MiddleName { get; set; }
    //public string LastName { get; set; }
    //public string Suffix { get; set; }
    [EditLink, Width(250)]
    public string FullName { get; set; }


}

File: 2025/Company/Company.Web/Modules/ServerTypes/Company/BusinessEntityColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { BusinessEntityRow } from "./BusinessEntityRow";

export interface BusinessEntityColumns {
    BusinessEntityId: Column<BusinessEntityRow>;
    FullName: Column<BusinessEntityRow>;
}

export class BusinessEntityColumns extends ColumnsBase<BusinessEntityRow> {
    static readonly columnsKey = 'Company.BusinessEntity';
    static readonly Fields = fieldsProxy<BusinessEntityColumns>();
}